docs/ispc.rst

```diff
@@ -886,6 +886,15 @@ New Architecture Support:
   Vector Extension (RVV) ISA. The new ``rvv-x4`` target provides 4-wide
   vectorization for RISC-V processors with vector extensions.
 
+Languages Changes:
+
+The compiler now assumes that all loops with non-constant conditions will make
```

Comment:
I had to read the implementation to understand what "non-constant conditions" really means here. Which means it needs to be explained better.